packages/sil-governance/src/routes/dashboard/components/Proposal/Period.tsx

```diff
@@ -3,22 +3,22 @@ import React from "react";
 
 interface Props {
   readonly expiryDate: Date;
-  readonly creationDate: Date;
+  readonly startDate: Date;
   readonly hasEnded: boolean;
 }
 
-const Period = ({ expiryDate, creationDate, hasEnded }: Props): JSX.Element => {
+const Period = ({ expiryDate, startDate, hasEnded }: Props): JSX.Element => {
   const expiryLabel = hasEnded ? "Expired on" : "Expires on";
   const expiryLocaleDate = expiryDate.toLocaleString();
-  const creationLocaleDate = creationDate.toLocaleString();
+  const startLocaleDate = startDate.toLocaleString();
 
   return (
     <Block display="flex" alignItems="center">
       <Typography variant="body2" weight="semibold">
         {expiryLabel} {expiryLocaleDate}
       </Typography>
       <Block marginLeft={1}>
-        <Typography variant="body2">Created on {creationLocaleDate}</Typography>
+        <Typography variant="body2">Created on {startLocaleDate}</Typography>
```

Comment:
Do I understand correctly this is the date when the voting on the proposal starts? Then the proposal can be created a long time before the voting actually starts. So I suggest the texts "Starts on" / "Started on" like we have with `expiryLabel`.

Sorry if this is out of scope for this PR. You can also create a different ticket if you want to merge as is

Reply:
Yes, the proper creation date of the proposal doesn't belong in the UI, I believe. The wireframes called the "start date" "creation date". I'll update the label no problem.

Reply:
Creation date of the proposal is not important for now. But the governor needs to know when they can vote.